config/config.yml

```diff
@@ -34,6 +34,7 @@ RETURN_ADDRESS: 'do-not-reply@example.org'
 SPAM_ALERT_ADDRESS: 'abuse-discuss@example.org'
 SPAM_THRESHOLD: 15
 ADMIN_ADDRESS: 'admin@example.org'
+TAG_WRANGLER_SUPERVISORS_ADDRESS: 'tagwranglers-personnel@transformativeworks.org'
```

Comment:
Should this also have `@example.org` like the other default addresses?

Reply:
Yes please! We prefer placeholder values in the config file.